docs/_pages/configuring.markdown

```diff
@@ -10,101 +10,77 @@ description: How to configure TRIRIGA Indoor Maps for your IBM TRIRIGA solution.
-triBuilding | Esri Building Default Rotation (triEsriRotationNU) | New field. See installation instructions for adding.  Depends on the shape of the building and the most common screen format people will view it in.  If the building is wider East to West it probably makes sense to rotate it 90 degrees.  Experiment with it to find the best value for the most common medium.
+triBuilding | GIS Latitude (triGisLatitudeNU) | This is an existing field.  It can be easily obtained by googling the address and looking in the URL. You may have to tweak the link to get to the preferred position.
+triBuilding | GIS Longitude (triGisLongitudeNU) | This is an existing field.  It can be easily obtained by googling the address and looking in the URL. You may have to tweak the link to get to the preferred position.
+triBuilding | Esri Map ID (triEsriMapIdTX) | this is a new field. For adding, refer to the installation instructions. See the following instructions for configuring. (Ex. 403b6012276b4339af152e9c3319a4b1)
+triBuilding | Esri Portal URL (triEsriPortalUrl) | This ia a new field. For adding, refer to the installation instructions. The URL is determined during Esri installation. (Ex. https://host.domain/portal)
+triBuilding | Esri Network URL (triEsriNetworkUrl) | This is a new field. For adding, refer to the installation instructions. See the following instructions for configuring. 
+triBuilding | Esri Building ID (triEsriBuildingID) | This is new field. For adding, refer to the installation instructions.  See the following instructions for configuring. (Ex. NC_CHARLOTTE_ONE)
+triFloor | Esri Floor ID (triEsriFloorID) | This is a new field. For adding, refer to the installation instructions.  See the following instructions for configuring. (Ex. NC_CHARLOTTE_ONE.1)
+triBuilding | Esri Building Default Zoom (triEsriZoomNU) | This is a new field. For adding, refer to the installation instructions.  Depends on the footprint of the building, 20 is probably ok for most buildings, however, needs to be experimented with.
+triBuilding | Esri Building Default Rotation (triEsriRotationNU) | This is a new field. For adding, refer to the installation instructions.  Depends on the shape of the building and the most common screen format people will view it in.  If the building is wider East to West, it probably makes sense to rotate it 90 degrees.  Experiment with it to find the best value for the most common medium.
 
 
 
 ## Obtaining Esri Map ID
 
-1. Open a web browser.
-1. Navigate to your Esri installation's Portal e.g. `https://host.domain/portal`
-1. Click the `Sign In` on the top right of the landing page.
+1. Navigate to your Esri installation's portal, e.g. `https://host.domain/portal`.
+2. Click `Sign In` on top right of the landing page.
```

Comment:
Markdown automatically numbers the numbered list correctly so you don't have to.  That's one feature of markdown that makes it more editor-friendly.  Replace all the numbers with 1.

Reply:
Fix this in all the other files too.